conf/modules.config

```diff
@@ -62,8 +62,13 @@ process {
         publishDir = [ enabled: false ]
     }
     withName: 'BOWTIE2_BUILD' {
+        // Check if the Bowtie2 index exists and skip if it does
+        process.when = !new File(params.bowtie2_db + '.1.bt2').exists()
```

Comment:
This doesn't work. `process.when` is not a directive. Also conditional execution should be included in the code to make it transparent.

Reply:
Okay, I see. I still received an error about bowtie2-build when trying to run with the suggestions you made in this PR, that's why I came up with another suggestion.

Reply:
If you have something I can test on, I can make sure it works.